python/stempy/image/__init__.py

```diff
@@ -1,9 +1,38 @@
 from stempy import _image
+from stempy import _io
 import numpy as np
-
-def create_stem_images(reader, inner_radii,
+import h5py
+
+
+def get_hdf5_reader(h5file):
+    # the initialization is at the io.cpp
+    dset_frame=h5file['frames']
+    dset_frame_shape=dset_frame.shape
+    totalImgNum=dset_frame_shape[0]
+    
+    dset_stem_shape=h5file['stem/images'].shape
+    scanwidth=dset_stem_shape[1]
+    scanheight=dset_stem_shape[2]
+    
+    # the number here are same with the smallscan data
+    blocksize=32
+    blockNumInFile=25
```

Comment:
Is number of blocks in file specific this dataset?

Reply:
yeah, since the small scan organize the data set like this, if the number is changed, there might be some problems. since the original image.cpp is programmed based on original smallscan data, I tried blocksize=16 blockNumInFile=50 , the outputted image is not correct.